### PR TITLE
lib/pf4: Fix radio and check alignment

### PR DESCRIPTION
**Pull request by garrett**
Fixes one of the issues in #15362

### CODE REVIEW

File: pkg/lib/patternfly/patternfly-4-overrides.scss [hunk 1]
```diff
@@ -76,3 +76,23 @@ svg {
       }
   }
 }
+
+// PF assumes check sizes are Chrome-based; size needs to be set for cross-browser compatibility
```
Comment (KKoukiou) [inline pkg/lib/patternfly/patternfly-4-overrides.scss:80]: As always the issue needs to be reported upstream. Of course not the fix, but at least issue with screenshots from the docs which prove it's not our issue.

**Comment by garrett** [comment]
- [x] Radios are not aligned to the middle of the first line of text (PF offset is incorrect)
- [x] Radios are inconsistently aligned across all browsers
- [x] Radios from PF3 clash with PF4
- [x] Checks are not aligned to the middle of the first line of text (PF offset is incorrect)
- [x] Checks are inconsistently aligned across all browsers
- [x] Checks from PF3 clash with PF4

File: pkg/lib/patternfly/patternfly-4-overrides.scss (hunk 1 above)
Comment (garrett) [reply pkg/lib/patternfly/patternfly-4-overrides.scss:80]: This is reported upstream, with the fix.

I forgot to add the comment in this PR; it's in the issue. I'll push with changes in a minute.

**Comment by garrett** [comment]
Upstream issue: https://github.com/patternfly/patternfly/issues/3868

This PR only fixes the alignment of radios and checks relative to their labels (not form labels).

File: pkg/lib/patternfly/patternfly-4-overrides.scss [hunk 2]
```diff
@@ -76,3 +76,25 @@ svg {
...
+// Alignment of checks is incorrect and inconsistent across browsers
+// Upstream issue: https://github.com/patternfly/patternfly/issues/3868
+.pf-c-radio > .pf-c-radio__input {
+    // Set the height of the input widget to be the same as its label
+    --pf-c-radio__input--Height: var(--pf-c-radio__label--FontSize);
+    // Offset by the top half of the difference between the height and the line height
+    --pf-c-radio__input--MarginTop: calc((var(--pf-c-radio__label--LineHeight) - 1) / 2 * var(--pf-c-radio__input--Height));
+    // PF3 sets a margin which messes with PF4, so set PF4's rule again here (Cockpit-specific mix of PF3+PF4)
+    margin-top: var(--pf-c-radio__input--MarginTop);
+}
```
Comment (garrett) [inline pkg/lib/patternfly/patternfly-4-overrides.scss:100]: Oops. I removed the last line. Will add it back in a force push.

**Review by skobyda — APPROVED** [review]
LGTM

**Pull request merged**